<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumWidget</class>
 <widget class="QWidget" name="SpectrumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="PlotWidget" name="graphWidget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>881</width>
     <height>521</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removePlotButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>60</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPlotButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>20</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPlotButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear all</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
